monix-eval/shared/src/main/scala/monix/eval/Task.scala

```diff
@@ -3871,7 +3929,11 @@ private[eval] abstract class TaskInstancesLevel1 extends TaskInstancesLevel0 {
     * {{{
     *   import cats.syntax.all._
     *
-    *   (taskA, taskB, taskC).parMap { (a, b, c) =>
+    *   val taskA = Task("a")
+    *   val taskB = Task("b")
+    *   val taskC = Task("c")
```

Comment:
See my comment above regarding using `sleep`. Also maybe add a note that it would complete in about 3 seconds, if 3 is your longest delay.